composer.json

```diff
@@ -6,7 +6,7 @@
     "license": "MIT",
     "require": {
         "php": "^8.0",
-        "sylius/sylius": "~1.12.0 || ~1.13.0",
+        "sylius/sylius": "~1.12.0 || ~1.13.0 || ~1.14.0",
```

Comment:
Hi @lruozzi9
Thanks for your contribution but could you please update the gh actions as well ? 😄
https://github.com/BitBagCommerce/SyliusProductBundlePlugin/tree/master/.github/workflows